### PR TITLE
Add integration tests for kopia utilities

### PR DESCRIPTION
**Pull request by ankitjain235**
## Change Overview

This PR adds integration tests for kopia utilities

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues 

- fixes #issue-number

## Test Plan

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: pkg/testing/cmd_test.go [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+		},
+		Hostname:        "test-hostname",
+		Username:        "test-username",
+		RepoPathPrefix:  fmt.Sprintf("test-path/test-%v", time.Now().Unix()),
+		ContentCacheMB:  0,
+		MetadataCacheMB: 0,
+		CacheDirectory:  "/tmp/cache",
+		Location:        locSecret.Data,
+	})
+	c.Assert(err, check.IsNil)
```
Comment (PrasadG193) [inline pkg/testing/cmd_test.go:102]: Just to add an extra validation, Is there a way to validate the status of the repository after creation?

File: pkg/testing/cmd_test.go [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+	accessKeyID := os.Getenv("AWS_ACCESS_KEY_ID")
+	if accessKeyID == "" {
+		c.Log("AWS_ACCESS_KEY_ID not set")
+		c.Fail()
+	}
+	secretAccessKey := os.Getenv("AWS_SECRET_ACCESS_KEY")
+	if secretAccessKey == "" {
+		c.Log("AWS_SECRET_ACCESS_KEY not set")
+		c.Fail()
+	}
```
Comment (PrasadG193) [inline pkg/testing/cmd_test.go:174]: Should we check if the Creds are set in the setup suite and skip the test?

**Review by PrasadG193 — COMMENTED** [review]
Left a few minor comments. Looks good overall

File: pkg/testing/cmd_test.go (hunk 1 above)
Comment (ankitjain235) [reply pkg/testing/cmd_test.go:102]: Yes, we can run `repository status` command to check the status. Also, we can run `repository connect` to verify repository is created using expected credentials only. Will add those steps as well once the parent PRs are approved.

**Comment by pavannd1** [comment]
@ankitjain235 Do we need this? What is remaining?

**Comment by ankitjain235** [comment]
@pavannd1 I need to add more tests to it, around repository connect etc. Will do that.

**Comment by ankitjain235** [comment]
Created https://github.com/kanisterio/kanister/pull/1955, closing this due to stale things.

**Pull request closed without merging**